src/cloffeine/async_cache.clj

```diff
@@ -1,41 +1,46 @@
 (ns cloffeine.async-cache
   (:refer-clojure :exclude [get])
-  (:require [cloffeine.common :as common]
-            [cloffeine.cache :as cache])
+  (:require [cloffeine.cache :as cache]
```

Comment:
this is a redundant change.

Reply:
I like to sort requires in this order:

1. Clojure
2. 3rd party libs
3. Internal libs
4. Service namespaces

And in each category to sort lexicographically. You get this in IntelliJ when editing Java code ("Optimize Imports"). In Clojure you can configure this with a linter rule to `clj-kondo` if my memory serves me right.